draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -2444,22 +2453,23 @@ Internet-Draft                 ACTN POI                       July 2023
 5.2. Multi-layer IP Link Setup
 
    To setup a new multi-layer IP link between two router ports, the MDSC
-   requires the O-PNC to setup an optical tunnel (either a WSON Tunnel
-   or a Flexi-grid Tunnel or an OTN Tunnel) within the optical network
-   between the two TTPs associated, as described in section 5.1, with
-   these two router Ethernet interfaces.
+   requires the O-PNC to setup an optical tunnel (either a WDM Tunnel or
+   an OTN Tunnel) within the optical network between the two TTPs
+   associated, as described in section 5.1, with these two router
+   Ethernet interfaces.
```

Comment:
In section 5.1 we talk about association between TTP with the router ports. Maybe better to maintain the same relationship and not using Ethernet interfaces.

Reply:
The term router port represents a physical entity so it is better to avoid this in the description of the abstract topology: Rephrased a bit the text in section 5.1 and in this section to avoid the term router port.